apps/spruce/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "bootstrap-logkeeper": "./scripts/bootstrap-logkeeper.sh",
+    "bootstrap-s3-logs": "../../scripts/bootstrap-s3-logs.sh",
```

Comment:
Yarn commands can be called from anywhere within the Spruce directory and it would be nice to support this rather than failing due to a relative path; would using the `--cwd` path allow that to work?

Reply:
--cwd will also adjust the cwd referenced in the script which I think is harder to debug. I ran into that during this [PR](https://github.com/evergreen-ci/ui/pull/135)